pkgs/virtualbox/default.nix

```diff
@@ -0,0 +1,206 @@
+{ stdenv, fetchurl, lib, iasl, dev86, pam, libxslt, libxml2
+, libIDL, libcap, zlib, libpng, glib, kernel, lvm2
+, which, alsaLib, curl, libvpx, gawk, nettools
+, xorriso, makeself, perl, pkgconfig, nukeReferences
+, javaBindings ? false, jdk ? null
+, pythonBindings ? false, python ? null
+, enableExtensionPack ? false, requireFile ? null, patchelf ? null, fakeroot ? null
+, pulseSupport ? false, pulseaudio ? null
+, guiSupport ? true, xproto ? null, libX11 ? null, libXext ? null, libXcursor ? null, qt4 ? null, SDL ? null, libXmu ? null
+, enableHardening ? false
+}:
+
+with stdenv.lib;
+
+let
+  buildType = "release";
+
+  version = "4.3.20"; # changes ./guest-additions as well
```

Comment:
You're right, and I don't think there's much interest in maintaining them here. Fixed in https://github.com/zalora/microgram/commit/bc18eb1eedb1d83037231fc9f2316b40154ce3b5.